### PR TITLE
Add decorates to screening view

### PR DESCRIPTION
**Pull request by katiejiang**
## Checklist
- [x] I ran `python manage.py format` to autoformat my code according to the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by katiejiang** [comment]
@ninatchang yes! Only admins can change the questions.

File: app/reminder/views.py [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
 def dashboard():
     """Dashboard to view and add text and email reminders."""
-    return render_template('reminder/index.html', reminders=Reminder.query.all())
+    return render_template(
+        'reminder/index.html', reminders=Reminder.query.all())
 
 
 @reminder.route('/new', methods=['GET', 'POST'])
+@login_required
+@admin_required
```
Comment (stephanieyshi) [inline app/reminder/views.py]: I think this should also include `@advisor_required`

**Comment by stephanieyshi** [comment]
Couldn't think of anything else off the top of my head—would just make sure to go through the PRD to double check as well 😊

File: app/reminder/views.py (hunk 1 above)
Comment (stephanieyshi) [reply app/reminder/views.py]: or i'm not sure if there will be a specific route for user specific reminders? @katiejiang

Comment (katiejiang) [reply app/reminder/views.py]: No user specific reminders

Comment (katiejiang) [reply app/reminder/views.py]: Don't think we need to worry too much about that

**Comment by katiejiang** [comment]
@ninatchang  To make sure your branch is up to date with all the latest changes, you'll need to merge master into your branch. Could you run:
```
git checkout master
git pull origin master
git checkout <your-branch-name>
git merge master
```

**Pull request merged**